<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerLogin</class>
 <widget class="QWidget" name="ServerLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>110</y>
     <width>200</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;SERVER CREATOR&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
   </property>
   <property name="title">
    <string>SERVER CREATOR</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_1">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;IP-ADDRESS:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>PASSWORD:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passswordLine">
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 0, 0);
font: 75 9pt &quot;MS Shell Dlg 2&quot;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;IP-ADDRESS:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>IP-ADDRESS:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ipLine">
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 0, 0);
font: 75 9pt &quot;MS Shell Dlg 2&quot;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;PORT:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>PORT:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portLine">
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 0, 0);
font: 75 9pt &quot;MS Shell Dlg 2&quot;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
